gr-qtgui/grc/qtgui_range.block.yml

```diff
@@ -76,7 +86,9 @@ templates:
             range = 'self._%s_range'%id
         %>\
         ${range} = Range(${start}, ${stop}, ${step}, ${value}, ${min_len})
-        ${win} = RangeWidget(${range}, self.set_${id}, "${no_quotes(label,repr(id))}", "${widget}", ${rangeType}, ${orient})
+        ${win} = GrRangeWidget(${range}, self.set_${id}, "${no_quotes(label,repr(id))}", "${widget}", ${rangeType}, ${orient}, "${outputmsgname}")
```

Comment:
You must change "${outputmsgname}" to "${no_quotes(outputmsgname)}" otherwise the key is quoted and will not be recognized by the receiver